package main

import "testing"

func TestApartmentApplicationUseCase(_ *testing.T) {
	// If there is an error in main this test will fail
	main()
}
